tools/manifest-schema/src/lib.rs

```diff
@@ -0,0 +1,385 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+#![allow(clippy::missing_panics_doc, clippy::too_long_first_doc_paragraph)]
+
+use std::{
+    cmp::{self, Reverse},
+    collections::BTreeMap,
+    fmt,
+    path::Path,
+    slice,
+    str::FromStr,
+};
+
+use anyhow::Result;
+use serde::{
+    de::{self, Deserialize, Deserializer},
+    ser::{Serialize, Serializer},
+};
+use serde_derive::{Deserialize, Serialize};
+
+pub const MANIFEST_SCHEMA_BRANCH_NAME: &str =
+    concat!("manifest-schema-", env!("CARGO_PKG_VERSION_MINOR"));
```

Comment:
Branch name here is `manifest-schema-<minor>`,

Reply:
Thanks, updated